Shopping List/ShoppingListController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ShoppingListController.swift
+//  Shopping List
+//
+//  Created by Alex on 5/3/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+
+class ShoppingListController {
+    var shoppingList: [Shopping] = []
+    let userDefaults = UserDefaults.standard
+    
+    private var persistentURL: URL? {
+        let fileManager = FileManager.default
+        guard let documentDir = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first else {return nil}
+        return documentDir.appendingPathComponent("shoppingList.plist")
+    }
+    
+    init() {
+        create() // not creating values - being called in wrong place?
+        loadFromPersistentStore()
+    }
+
+    func create(){
+        let itemNames = ["apple", "grapes", "milk", "muffin", "popcorn", "soda", "strawberries"]
+        for item in itemNames {
+            shoppingList.append(Shopping(imageName: item, itemName: item))
+            saveToPersistentStore()
+        }
+    }
+    
+    func toggleAdd(shoppingListItem: Shopping) {
+        guard let index = shoppingList.index(of: shoppingListItem) else { return }
+        shoppingList[index].addedToList.toggle()
+        
```

Comment:
You might want to think about implementing some logic to deal with item count here. As it is now, Item count is always 7 since that is the size of the array in your persistent store. a ```addedItemCount``` would be fairly straightforward to implement.